plugins/chunter-impl/src/components/ThreadsView.svelte

```diff
@@ -47,28 +62,107 @@
     })
   }
 
-  function addMessage (text: string): void {
-    client.createDoc(chunter.class.Comment, message!.space, {
+  async function addMessage (text: string): Promise<void> {
+    await client.createDoc(chunter.class.Comment, message!.space, {
       replyOf: getFullRef(message!._id, message!._class),
       message: text
     })
+    lastTime = Date.now()
+    await updateLastRead(message!)
+  }
+
+  beforeUpdate(async () => {
```

Comment:
Again very similar logic, could we have some generic code for auto scrolling to Messages?